client/client_test.go

```diff
@@ -691,7 +691,7 @@ func (ks *serverKeyStore) AuthRole(authid string) (string, error) {
 
 func TestConnectContext(t *testing.T) {
 	const (
-		expect     = "dial tcp: lookup localhost: operation was canceled"
+		expect     = "dial tcp: operation was canceled"
```

Comment:
I think the better way to fix this is to either change CI to use a later version of go, or have the test look for "operation was canceled"